src/nft/types/common/common.ts

```diff
@@ -93,17 +94,17 @@ export interface GenieAsset {
   sellorders?: SellOrder[]
   smallImageUrl?: string
   tokenId: string
-  tokenType: TokenType
+  tokenType?: NftStandard
```

Comment:
do you think using erc721 as the default token type would be a bad idea?

Reply:
Yeah I think it's better to be undefined in this case as per https://github.com/Uniswap/interface/pull/5665#discussion_r1048978571